code/src/index.css

```diff
@@ -11,3 +11,148 @@ code {
   font-family: source-code-pro, Menlo, Monaco, Consolas, "Courier New",
     monospace;
 }
+
+/*Entire site styling*/
+.site-container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+/*Form styling*/
+.form-container {
+  background-color: rgb(243, 242, 242);
+  border: solid 1px black;
+  width: 300px;
+  padding-top: 10px;
+  margin-top: 15px;
+  box-shadow: 10px 10px;
+}
+
+.thought-form {
+  display: flex;
+  flex-direction: column;
+}
+
+.form-text {
+  font-size: 14px;
+  margin: 10px;
+}
+
+.input-bar {
+  margin: 10px;
+  padding: 10px;
+  width: 250px;
+}
+
+.send-button {
+  border: none;
+  background: rgb(248, 205, 213);
+  margin: 10px;
+  border-radius: 20px;
+  width: 70%;
+}
```

Comment:
<img width="353" alt="Screenshot 2021-03-30 at 12 42 07" src="https://user-images.githubusercontent.com/50805742/112977493-6b661800-9156-11eb-86ce-168cd64f42e5.png">

when I check your website fullscreen, the button gets very big with a lot of empty space between the text and the button boarder. I would suggest making the width of the button smaller :)